### PR TITLE
CP-23450: Add "View" btn in updates page toolbar

### PR DESCRIPTION
**Pull request by jijiang**
The View button contains "By Update" and "By Host" options.

### CODE REVIEW

File: XenAdmin/TabPages/ManageUpdatesPage.Designer.cs [hunk 1]
```diff
@@ -423,5 +467,10 @@ private void InitializeComponent()
         private System.Windows.Forms.DataGridViewTextBoxColumn ColumnDate;
         private System.Windows.Forms.DataGridViewTextBoxColumn ColumnWebPage;
         private System.Windows.Forms.TableLayoutPanel tableLayouPanel5;
+        private System.Windows.Forms.ToolStripSeparator toolStripSeparator;
+        private System.Windows.Forms.ToolStripSplitButton toolStripSplitButtonView;
```
Comment (kc284) [inline XenAdmin/TabPages/ManageUpdatesPage.Designer.cs]: The control for this use should be ToolStripDropDownButton, not ToolStripSplitButton. With the ToolStripSplitButton the user selects an item and this item becomes the default, so next time they want to repeat the action they can click it directly without having to open the dropdown again. However, here you only select one of the two items, you don't need to repeat an action, therefore you need the ToolStripDropDownButton.

Also, when you select one of the dropdown items, this item should stay highlighted. Currently, the first item is always highlighted, no matter which one you click.

(We have a similar button on the Snapshots TabPage, you can have a look there to see how it works).

File: XenAdmin/TabPages/ManageUpdatesPage.Designer.cs [hunk 2]
```diff
@@ -203,10 +210,42 @@ private void InitializeComponent()
...
+            this.byUpdateToolStripMenuItem.Checked = true;
+            this.byUpdateToolStripMenuItem.CheckState = System.Windows.Forms.CheckState.Checked;
+            this.byUpdateToolStripMenuItem.Image = global::XenAdmin.Properties.Resources.notif_updates_16;
+            this.byUpdateToolStripMenuItem.Name = "byUpdateToolStripMenuItem";
+            resources.ApplyResources(this.byUpdateToolStripMenuItem, "byUpdateToolStripMenuItem");
+            this.byUpdateToolStripMenuItem.Click += new System.EventHandler(this.byUpdateToolStripMenuItem_Click);
+            // 
+            // byHostToolStripMenuItem
+            // 
+            this.byHostToolStripMenuItem.Image = global::XenAdmin.Properties.Resources._000_TreeConnected_h32bit_16;
```
Comment (kc284) [inline XenAdmin/TabPages/ManageUpdatesPage.Designer.cs]: The images look a bit stretched and blurry, you probably need to try the other scaling options in their properties.

**Comment by kc284** [comment]
I understand it's a bit difficult to split this big chunk of work into items because they depend on each other, but I think we can put a bit more code in this PR, namely hide/show the relevant buttons in the top toolstrip when the View changes.

**Comment by kc284** [comment]
One more thing: because this is a biggish piece of work and it will be at least 3 tickets/PRs before it is completely functional, I recommend you close this PR and re-submit it to the freshly created branch REQ-534, because the master should in theory be releasable at all times.

**Comment by jijiang** [comment]
As recommended by Tina, this change should be part of REQ-534, and should be pushed to REQ-534 branch instead.

**Pull request closed without merging**